VMEncryption/main/KeyVaultUtil.py

```diff
@@ -226,7 +226,7 @@ def create_secret(self, AccessToken, KeyVaultURL, secret_value, KeyEncryptionAlg
 
             self.logger.log("{0} {1}".format(result.status, result.getheaders()))
             result_content = result.read()
-            self.logger.log("result_content is {0}".format(result_content))
+            # self.logger.log("result_content is {0}".format(result_content))
```

Comment:
Remove the line

Reply:
And maybe add a comment to not log the result in future since it contains secret

---
In reply to: [244847482](https://github.com/Azure/azure-linux-extensions/pull/711#discussion_r244847482) [](ancestors = 244847482)